allennlp_models/generation/models/copynet_seq2seq.py

```diff
@@ -68,6 +68,8 @@ class CopyNetSeq2Seq(Model):
         in the batch and the second is a gold sequence for each item in the batch.
     initializer : `InitializerApplicator`, optional
         An initialization strategy for the model weights.
+    target_decoder_layers : `int`, optional (default = `1`)
```

Comment:
I think the `target` in `target_decoder_layers` might be unnecessary? Maybe `num_decoder_layers` is more informative?

Reply:
I agree ✋

Reply:
done!